src/Dfc.CourseDirectory.WebV2/Features/Apprenticeships/FindStandard.cs

```diff
@@ -43,49 +33,33 @@ public class ViewModel
         public IReadOnlyCollection<ViewModelResult> Results { get; set; }
     }
 
-    public class SelectCommand : IRequest<Success>
-    {
-        public StandardOrFramework StandardOrFramework { get; set; }
-    }
-
     public class ViewModelResult
     {
         public string ApprenticeshipTitle { get; set; }
         public bool IsFramework { get; set; }
-        public bool? OtherBodyApprovalRequired { get; set; }
+        public bool OtherBodyApprovalRequired { get; set; }
         public string NotionalNVQLevelv2 { get; set; }
-        public int? FrameworkCode { get; set; }
-        public int? FrameworkProgType { get; set; }
-        public int? FrameworkPathwayCode { get; set; }
-        public int? StandardCode { get; set; }
-        public int? StandardVersion { get; set; }
+        public int StandardCode { get; set; }
+        public int StandardVersion { get; set; }
     }
 
     public class Handler :
         IRequestHandler<Query, ViewModel>,
         IRestrictProviderType<Query>,
         IRequestHandler<SearchQuery, QueryResponse>,
-        IRestrictProviderType<SearchQuery>,
-        IRequestHandler<SelectCommand, Success>,
-        IRestrictProviderType<SelectCommand>
+        IRestrictProviderType<SearchQuery>
     {
         private readonly IStandardsAndFrameworksCache _standardsAndFrameworksCache;
```

Comment:
Can we rename/update `IStandardsAndFrameworksCache` yet or is there other stuff depending on it? Or is that another PR?

Reply:
Yeh there's still a few things that refer to it (e.g. bulk upload)